packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -73,6 +73,32 @@ export interface ConfigurationSetProps {
    * @default - use the default awstrack.me domain
    */
   readonly customTrackingRedirectDomain?: string;
+
+  /**
+   * The Virtual Deliverability Manager (VDM) options that apply to the configuration set
+   *
+   * @default - VDM options not configured at the configuration set level. In this case, use account level settings.
```

Comment:
Sorry, I should have made myself clearer. I think it's more important to tell the users that by default, VDM options are disabled, unless there are account level settings. Your current documentation only states that if they happen to have account level settings, those will be used. I would expect most of the CDK users will not be using account level settings.